crates/server/src/builder.rs

```diff
@@ -101,6 +103,7 @@ impl<S: AsRef<Path>> Builder<S> {
             make_service: Mutex::new(
                 Router::new()
                     .fallback(handle.into_service())
+                    .route("/health", get(|| async { StatusCode::OK }))
```

Comment:
```suggestion
                    .route("/health", get(|| async {}))
```

doesn't this suffice?